<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="1121" started-at="2017-03-04T22:22:59Z" finished-at="2017-03-04T22:23:00Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="1121" started-at="2017-03-04T22:22:59Z" finished-at="2017-03-04T22:23:00Z">
      <class name="models.DomainTests">
        <test-method status="FAIL" signature="accountUpdateRecord()[pri:0, instance:models.DomainTests@6737fd8f]" name="accountUpdateRecord" duration-ms="1110" started-at="2017-03-04T17:22:59Z" finished-at="2017-03-04T17:23:00Z">
          <exception class="java.lang.ClassCastException">
            <message>
              <![CDATA[com.tinkerpop.frames.structures.FramedVertexIterable cannot be cast to java.util.List]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.ClassCastException: com.tinkerpop.frames.structures.FramedVertexIterable cannot be cast to java.util.List
	at com.sun.proxy.$Proxy5.getGroups(Unknown Source)
	at models.DomainTests.accountUpdateRecord(DomainTests.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.ClassCastException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- accountUpdateRecord -->
      </class> <!-- models.DomainTests -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
